### PR TITLE
[NETBEANS-2660] - Added javadoc for java 9, 10, 11, 12 & 13 when addi…

### PR DESCRIPTION
**Pull request by pepness**
…ng a new java platform

### CODE REVIEW

File: java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/platformdefinition/J2SEPlatformDefaultJavadocImpl.java [hunk 1]
```diff
@@ -57,7 +57,11 @@
         OFFICIAL_JAVADOC.put("1.6", "https://docs.oracle.com/javase/6/docs/api/"); // NOI18N
         OFFICIAL_JAVADOC.put("1.7", "https://docs.oracle.com/javase/7/docs/api/"); // NOI18N
         OFFICIAL_JAVADOC.put("1.8", "https://docs.oracle.com/javase/8/docs/api/"); // NOI18N
-        OFFICIAL_JAVADOC.put("9", null); //Does not exist for now
+        OFFICIAL_JAVADOC.put("9", "https://docs.oracle.com/javase/9/docs/api/"); // NOI18N
+        OFFICIAL_JAVADOC.put("10", "https://docs.oracle.com/javase/10/docs/api/"); // NOI18N
+        OFFICIAL_JAVADOC.put("11", "https://docs.oracle.com/en/java/javase/11/docs/api/"); // NOI18N
+        OFFICIAL_JAVADOC.put("12", "https://docs.oracle.com/en/java/javase/12/docs/api/"); // NOI18N
+        OFFICIAL_JAVADOC.put("13", "https://download.java.net/java/early_access/jdk13/docs/api/"); // Early access
```
Comment (junichi11) [inline java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/platformdefinition/J2SEPlatformDefaultJavadocImpl.java]: Please add `NOI18N`. e.g. `// NOI18N Early access`

**Comment by junichi11** [comment]
The Travis CI fails: Please try merging the latest master branch( or rebasing  with the latest master branch), then try pushing again. (if you rebase your branch, need force option `-f`)

File: java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/platformdefinition/J2SEPlatformDefaultJavadocImpl.java (hunk 1 above)
Comment (pepness) [reply java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/platformdefinition/J2SEPlatformDefaultJavadocImpl.java]: Thanks for the review Junuchi, will add `NOI18N`.

**Comment by pepness** [comment]
I did not squash the commits into a single one, should I close this PR and create another? After all this PR would be merged to NB 11.2 and it will have a cleaner history.

**Comment by junichi11** [comment]
> I did not squash the commits into a single one, should I close this PR and create another?

Please don't worry. We can squashing the PR via the "Squash and merge" button. Of course, you can squash them yourself, then you can also force-push.  So need not to close this PR :)

Thanks for fixing it. We cannot merge this into the master branch in the current phase(we can merge only PRs related to NB11.1, currently). So please wait a while. Thanks!

**Comment by neilcsmith-net** [comment]
IMO, this is a good candidate for merging for NB 11.1. Has a JIRA ticket, and fixes a bug.  https://issues.apache.org/jira/browse/NETBEANS-2660

**Comment by junichi11** [comment]
@neilcsmith-net You are the release manager. So I think you can merge this PR if it's OK with you.

**Comment by neilcsmith-net** [comment]
@junichi11 just giving anyone a chance to disagree! Right, squashing and merging.

**Pull request merged**